docs/en/observability/ingest-splunk.asciidoc

```diff
@@ -0,0 +1,131 @@
+[chapter, role="xpack"]
+[[ingest-splunk]]
+
+:modulename: system nginx mysql
+
+[[ingest-splunk]]
+= Ingest data from Splunk (Experimental)
+
+[NOTE]
+=====
+If you haven't already, you need to install {es} for storing and
+searching your data, and {kib} for visualizing and managing it. For
+more information, see <<add-observability-data,Send data to {es}>>.
+After {es} and {kib} are installed Fleet must be enabled, see
+https://www.elastic.co/guide/en/fleet/current/fleet-quick-start.html[Fleet Quick Start].
+
+
+=====
+
+Apache, AWS Cloudtrail, Nginx, and Zeek integrations offer the ability
+to seamlessly ingest data from a Splunk Enterprise instance.  Data
+will be automatically mapped to the Elastic Common Schema, making it
+available for rapid analysis in Elastic solutions, including Security
+and Observability.
+
+These integrations work by using the `httpjson` input in {agent} to
+run a Splunk search via the Splunk REST API and then extract the raw
+event from the results.  The raw event is then processed via the
+{agent}.  The Splunk search is customizable and the interval between
+searches is customizable.  These integrations only get new data since
+the last query, not historical data.
+
+[role="screenshot"]
+image::images/elastic-agent-splunk.png[Splunk integration components]
+
+To ingest Nginx data from Splunk the following steps are performed.
+The options are the same for Apache, AWS Cloudtrail and Zeek.
+
+== Step 1: Add integration
```

Comment:
We like to explicitly name anchors because Asciidoc doesn't always auto-generate great ones :) 

```suggestion
[discrete]
[[splunk-step-one]]
== Step 1: Add integration
```